app/lib/publisher/backend.dart

```diff
@@ -217,6 +217,20 @@ class PublisherBackend {
     });
   }
 
+  /// The list of e-mail addresses of the members with admin roles. The list
+  /// should be used to notify admins on upload events.
+  Future<List<String>> getAdminMemberEmails(String publisherId) async {
+    final result = <String>[];
+    final key = _db.emptyKey.append(Publisher, id: publisherId);
+    final query = _db.query<PublisherMember>(ancestorKey: key)
+      ..filter('role =', PublisherMemberRole.admin);
+    await for (final m in query.run()) {
```

Comment:
Should we tweak `getEmailOfUserId` to accept a list of userIds and return a list of emails..
Wouldn't that be more efficient in terms of datastore operations?

Reply:
Good point, we already have a batching for it, switched to `getEmailsOfUserIds`. Under the hood it uses a cache too.